style.css

```diff
@@ -0,0 +1,62 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#img1 {
+  justify-content: flex-start;
+}
+
+.ic_icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex: 0;
+  padding: 8%;
+}
+
+#btn {
+  justify-content: space-between;
+  border-radius: 8%;
+  background: #1a2236;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.group48 {
+  display: flex;
+  flex-direction: column;
+}
+
+#title {
+  font-family: 'Courier New', Courier, monospace; /* Playfair Displa; */
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+#text {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+}
+
+#group19 {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  background: #1a2236;
+  flex-grow: 1;
+}
```

Comment:
- [ ] Kindly stick to the Figma design as much as possible. The button "Start collaboration" is a little different compared to the Figma design. Therefore, I suggest the following change to make it looks more attractive:
  - Kindly increase the `height` of the button to follow the Figma design
  - Kindly use white color for the button's text as show in the Figma design
  - Kindly add more bottom margin to the button to follow the Figma design
### Your design :point_down: 
![Screenshot from 2023-07-06 08-28-23](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/cc44f733-18af-4a52-b92b-36cbcebdcbd0)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-07-06 08-26-23](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/4f8753c2-ec13-429f-9867-371474994645)